apps/demo/src/app/examples/components-example/components-example.component.ts

```diff
@@ -1,15 +1,22 @@
 import { Component, OnInit } from '@angular/core';
 
+const img1 = '/assets/example_md.png';
```

Comment:
could you use the already existing file and copy that to the assets folder during the build using `angular.json` ?